src/components/ui-shell/_header.scss

```diff
@@ -8,94 +8,94 @@
 @import 'theme';
 @import 'functions';
 
-@mixin platform-header {
-  .#{$prefix}--platform-header {
+@mixin header {
```

Comment:
Wondering if we should make this mix-in name more specific to avoid mix-in name conflict (with ones in apps)?

Reply:
Added `carbon-` prefixes! 👍